### PR TITLE
Add byte string array and float128 max value to sample HDF5 file

### PR DESCRIPTION
**Pull request by axelboc**

### CODE REVIEW

File: packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap [hunk 1]
```diff
@@ -1745,8 +1769,8 @@ exports[`test file matches snapshot 1`] = `
           0,
           0,
           0,
-          80,
-          191,
+          72,
+          203,
```
Comment (axelboc) [inline packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap:1773]: I guess this is a side-effect of adding a new dataset to the file.

File: create_h5_sample.py [hunk 2]
```diff
@@ -113,6 +117,11 @@ def print_h5t_class(dataset):
     # === H5T_OPAQUE ===
 
     add_scalar(h5, "byte_string", np.void(b"\x00\x11\x22"))
+    add_array(
+        h5,
+        "byte_string",
+        np.array([np.void(b"\x00"), np.void(b"\x11"), np.void(b"\x22")]),
+    )
```
Comment (axelboc) [inline create_h5_sample.py:124]: Adding this as a sanity check in preparation for #1554

File: packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap (hunk 1 above)
Comment (loichuder) [reply packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap:1773]: Wait, what ? How adding new datasets changes the value of existing ones ?

Comment (axelboc) [reply packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap:1773]: I think those are basically buffer addresses in the file.

Comment (loichuder) [reply packages/h5wasm/src/__snapshots__/h5wasm-api.test.ts.snap:1773]: Oh got it :+1:

**Review by loichuder — APPROVED** [review]

**Pull request merged**